Transformers/Java/src/main/resources/config.properties

```diff
@@ -33,3 +33,4 @@ EmptyMethodTransformer=true
 EmptyMethodStringRandomness=pseudo
 AddUnusedVariableTransformer=true
 UnusedVariableStringRandomness=pseudo
+RenameVariableStringRandomness=pseudo
```

Comment:
I think this needs also to have the RenameVariableStringTransformer=True, no?

Reply:
Everything that has a random element to it should be pseudo I thought. Since we rename the variable to a random string I thought it should be pseudo, right?